app/code/Bulbulatory/Recomendations/Controller/Recommendations/Index.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace Bulbulatory\Recomendations\Controller\Recommendations;
+
+class Index extends \Magento\Framework\App\Action\Action
+{
+	protected $_pageFactory;
+
+	public function __construct(
+		\Magento\Framework\App\Action\Context $context,
+		\Magento\Framework\View\Result\PageFactory $pageFactory)
+	{
+		$this->_pageFactory = $pageFactory;
+		return parent::__construct($context);
+	}
+
+	public function execute()
+	{
+		return $this->_pageFactory->create();
```

Comment:
Jeśli moduł jest wyłączony, to nie powinniśmy pozwolić na wyświetlenie formularza. Powinniśmy też sprawdzić, czy klient jest zalogowany, bo tylko tacy powinni móc wysyłać rekomendacje - musimy przecież przypisać rekomendację do jakiegoś użytkownika, żeby później móc mu naliczać zniżki.